src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,131 @@
+package duke;
+
+import duke.task.*;
+
+import java.security.KeyPair;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+
+/** A class that manages the Tasks in the to-do list. */
+public class TaskList {
+
+    /** The list of tasks in the to-do list */
+    private static ArrayList<Task> taskArr;
+
+    public TaskList() {
```

Comment:
Not really required, but perhaps having JavaDoc comments for all public methods will make it more consistent.